python/ray/internal/internal_api.py

```diff
@@ -0,0 +1,54 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import ray.local_scheduler
+import ray.worker
+from ray import profiling
+import pyarrow.plasma as plasma
+
+__all__ = ["free"]
+
+
+def free(object_ids, spread=True, worker=ray.worker.get_global_worker()):
+    """Free a list of IDs from object stores.
+
+    This function is low-level API which should be used in restricted
+    scenarios.
+
+    If spread is set, the request will spread to all object stores.
+
+    This method will not return any value to indicate whether the deletion is
+    successful or not. This function is an instruction to object store. If
+    the some of the objects are in use, object stores will delete them later
+    when the ref count is down to 0.
+
+    Args:
+        object_ids (List[ObjectID]): List of object IDs to delete.
+        spread (bool): Whether deleting the list of objects in all object
```

Comment:
I have a slight preference to rename this param to `local_only`, which means if we only free the objects in local object store.

Don't know how you guys feel. But I feel `local_only` is more clear, I can understand the meaning without referencing the document.

But anyway, I don't have very strong preference. Up to you to choose the name.